apollo/src/types/Limit.ts

```diff
@@ -0,0 +1,4 @@
+export const LIMITS = [10, 25, 50] as const;
+
+type LimitTuple = typeof LIMITS;
```

Comment:
Sorry my brain is slow and dumb, could you ELI5 why not use an enum type instead? I'm just missing something obvious probably

Reply:
Sorry just got back home.

I actually didn't see your comments before (fun fact, you have to refresh 🙃). I was having a hard time getting the enums to play nicely with Pothos, but I didn't know you could define numeric enums. I'm gonna give that a shot now and repush to this PR.

Reply:
Okay, changed the `as const` to an actual enum. I kept the scalar and did not use a graphql enum because graphql enums cannot be referenced by their numeric values.

This means that the enum will resolve to the numeric value, but it cannot be referenced by its numeric value in the input. So if we had this enum:

```typescript
enum Limit {
    Ten = 10,
    TwentyFive = 25,
    Fifty = 50,
}
```

And we added this as an actual enum type in Pothos:

```typescript
builder.enumType(Limit, {
    name: 'Limit',
});
```

Then you would have to use, for example, `limit = 'Ten'` and not `limit = 10` in the query input.

Reply:
Okay, changed the `as const` to an actual enum. I kept the scalar and did not use a graphql enum because graphql enums cannot be referenced by their numeric values.

This means that the enum will resolve to the numeric value, but it cannot be referenced by its numeric value in the input. So if we had this enum:

```typescript
enum Limit {
    Ten = 10,
    TwentyFive = 25,
    Fifty = 50,
}
```

And we added this as an actual enum type in Pothos:

```typescript
builder.enumType(Limit, {
    name: 'Limit',
});
```

Then you would have to use, for example, `limit = 'Ten'` and not `limit = 10` in the query input.

Reply:
Ohhhhh weird and fun, cool cool